packages/mobile/src/components/lineup-tile/LineupTileMetadata.tsx

```diff
@@ -79,6 +82,12 @@ export const LineupTileMetadata = ({
     navigation.push('Profile', { handle: user.handle })
   }, [navigation, user])
 
+  const isPlaying = useSelector(getPlaying)
```

Comment:
I like pushing this logic into `LineupTileMetadata`!

But with this every `LineupTileMetadata` will rerender when the play state changes. With the combined selector it would only rerender when the play state changes and it is the actively playing tile:

```
  const isPlaying = useSelector(
    (state: CommonState) => getPlaying(state) && isPlayingUid
  )
  ```

Reply:
ahhhh makes sense! will fix ty